src/api_server/src/request/drive.rs

```diff
@@ -152,6 +157,48 @@ mod tests {
                 "path_on_host": "dummy"
               }"#;
         assert!(parse_patch_drive(&Body::new(body), Some(&"bar")).is_err());
+
```

Comment:
would be nice to have a comment for each of these test cases, like we do for the other assertions above, to catch the essence of the test case

Reply:
Added comments.